packages/contracts/src/contracts/current/protocol/Exchange/interfaces/IExchangeCore.sol

```diff
@@ -48,4 +48,18 @@ contract IExchangeCore {
     function cancelOrder(LibOrder.Order memory order)
         public
         returns (bool);
+
+    /// @dev Gets information about an order: status, hash, and amount filled.
+    /// @param order Order to gather information on.
+    /// @return status Status of order. Statuses are defined in the LibStatus.Status struct.
+    /// @return orderHash Keccak-256 EIP712 hash of the order.
+    /// @return takerAssetFilledAmount Amount of order that has been filled.
+    function getOrderInfo(LibOrder.Order memory order)
+        public
+        view
+        returns (
```

Comment:
Is there a reason this doesn't return an OrderInfo struct?

Reply:
That's a good idea. This struct was originally created as a workaround for the stack limit. But yeah, it totally makes sense to just return it rather than reinterpret the output.